test_examples.py

```diff
@@ -0,0 +1,26 @@
+import os, sys, importlib
+
+failed = []
+
+__verbose__ = 2
+for name in os.listdir( 'examples' ):
+  if not name.endswith( '.py' ):
+    continue
+  print( 'testing', name )
+  example = importlib.import_module( 'examples.'+name[:-3] )
+  for __nprocs__ in 1,2:
+    try:
+      example.unittest()
+    except Exception as e:
+      status = 'FAILED ({})'.format( e )
+      failed.append( '{} (nprocs={})'.format( name, __nprocs__ ) )
+    else:
+      status = 'OK'
+    print( '* nprocs={}: {}'.format( __nprocs__, status ) )
+
+if failed:
+  print( 'some tests failed:', ', '.join( failed ) )
+else:
+  print( 'all tests passed' )
+
+sys.exit( len(failed) )
```

Comment:
Indeed this is a placeholder, I wasn't sure if tests/ or inside setup.py would be best. The former then.